src/polkitbackend/polkitbackendauthority.c

```diff
@@ -1349,22 +1364,125 @@ server_handle_get_property (GDBusConnection  *connection,
+              break;
+            case LOG_LEVEL_WARNING:
+              result = g_variant_new_string ("warn");
+              break;
+            case LOG_LEVEL_NOTICE:
+              result = g_variant_new_string ("notice");
+              break;
+            case LOG_LEVEL_INFO:
+              result = g_variant_new_string ("info");
+              break;
+            case LOG_LEVEL_DEBUG:
+              result = g_variant_new_string ("debug");
+              break;
+            }
+        }
+      else if (g_strcmp0 (property_name, "LogTarget") == 0)
+        {
+          result = g_variant_new_string ("syslog");
+        }
+      else if (g_strcmp0 (property_name, "SyslogIdentifier") == 0)
+        {
+          result = g_variant_new_string ("polkitd");
+        }
+      else
+        g_assert_not_reached ();
+
     }
-  else if (g_strcmp0 (property_name, "BackendFeatures") == 0)
+
+  return result;
+}
+
+static gboolean
+server_handle_set_property (GDBusConnection  *connection,
+                            const gchar      *sender,
+                            const gchar      *object_path,
+                            const gchar      *interface_name,
+                            const gchar      *property_name,
+                            GVariant         *value,
+                            GError          **error,
+                            gpointer          user_data)
+{
+  PolkitSubject *caller_subject;
+  PolkitUnixUser *caller_user;
+  const gchar *level;
+
+  if (g_strcmp0 (interface_name, "org.freedesktop.LogControl1") != 0)
     {
-      result = g_variant_new_uint32 (polkit_backend_authority_get_features (server->authority));
+      g_set_error (error, G_IO_ERROR, G_IO_ERROR_INVALID_ARGUMENT, "Only properties of org.freedesktop.LogControl1 can be modified");
+      return FALSE;
     }
-  else
-    g_assert_not_reached ();
 
-  return result;
+  if (g_strcmp0 (property_name, "LogLevel") != 0)
+    {
+      g_set_error (error, G_IO_ERROR, G_IO_ERROR_INVALID_ARGUMENT, "Only LogLevel can be modified");
+      return FALSE;
+    }
+
+  caller_subject = polkit_system_bus_name_new (sender);
+  if (!caller_subject)
+    {
+      g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED, "Could not obtain caller's credentials");
+      return FALSE;
+    }
+  caller_user = polkit_system_bus_name_get_user_sync (POLKIT_SYSTEM_BUS_NAME (caller_subject), NULL, error);
+  if (!caller_user)
+    {
+      g_object_unref (caller_subject);
+      g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED, "Could not obtain caller's credentials");
+      return FALSE;
+    }
+  if ((uid_t)polkit_unix_user_get_uid (caller_user) != 0)
+    {
+      g_set_error (error, G_IO_ERROR, G_IO_ERROR_PERMISSION_DENIED, "Only root can change the log level");
+      g_object_unref (caller_user);
+      g_object_unref (caller_subject);
+      return FALSE;
+    }
+
+  g_variant_get (value, "&s", &level);
+  polkit_backend_authority_set_log_level (level);
```

Comment:
I think error handling is missing here in the sense that it accepts anything without returning errors. For example here's what `systemd-resolved` does:
```sh
# busctl set-property org.freedesktop.resolve1 /org/freedesktop/LogControl1 org.freedesktop.LogControl1 LogLevel "s" yolo
Failed to set property LogLevel on interface org.freedesktop.LogControl1: Invalid log level 'yolo'
# echo $?
1
```
Here's what polkit does:
```sh
# busctl set-property org.freedesktop.PolicyKit1 /org/freedesktop/LogControl1 org.freedesktop.LogControl1 LogLevel "s" yolo
# echo $?
0
```

Reply:
@jrybar-rh since it's still half-implemented should a new issue be opened to at least document how it interacts with `--no-debug` or is it going to be added to https://github.com/polkit-org/polkit/pull/425 (or any other PR) later?

Reply:
Or send pull request.

Reply:
Fair enough. I'll collect all the shortcomings and try to document them (though I'm not a native English speaker so I'm not the best person to write the documentation)